source/neuropod/multiprocess/multiprocess.cc

```diff
@@ -76,17 +76,18 @@ pid_t start_worker_process(const std::string &control_queue_name, std::vector<st
     // Setup the environment
 
     // Null terminated char * array
-    char *env_arr[env.size() + 1];
+    std::vector<char *> env_arr(env.size() + 1);
```

Comment:
in case of vector, all elements are initialized with 0 (nullptr) already because it calls (char*) init for every element, so next line is redundant (it was necessary for regular array though)